docs/src/main/sphinx/admin/resource-groups.md

```diff
@@ -171,15 +171,22 @@ documentation](https://docs.oracle.com/en/java/javase/22/docs/api/java.base/java
 
 - `queryType` (optional): string to match against the type of the query submitted:
 
-  - `SELECT`: `SELECT` queries.
-  - `EXPLAIN`: `EXPLAIN` queries (but not `EXPLAIN ANALYZE`).
-  - `DESCRIBE`: `DESCRIBE`, `DESCRIBE INPUT`, `DESCRIBE OUTPUT`, and `SHOW` queries.
-  - `INSERT`: `INSERT`, `CREATE TABLE AS`, and `REFRESH MATERIALIZED VIEW` queries.
-  - `UPDATE`: `UPDATE` queries.
-  - `DELETE`: `DELETE` queries.
-  - `ANALYZE`: `ANALYZE` queries.
-  - `DATA_DEFINITION`: Queries that alter/create/drop the metadata of schemas/tables/views,
-    and that manage prepared statements, privileges, sessions, and transactions.
+  - `SELECT`: [SELECT](/sql/select) queries.
+  - `EXPLAIN`: [EXPLAIN](/sql/explain) queries (but not [EXPLAIN ANALYZE](/sql/explain-analyze)).
+  - `DESCRIBE`: [DESCRIBE](/sql/describe), [DESCRIBE
+    INPUT](/sql/describe-input), [DESCRIBE OUTPUT](/sql/describe-output), and
+    `SHOW` queries such as [SHOW CATALOGS](/sql/show-catalogs), [SHOW
+    SCHEMAS](/sql/show-schemas), and [SHOW TABLES](/sql/show-tables).
+  - `INSERT`: [INSERT](/sql/insert), [CREATE TABLE AS](/sql/create-table-as), and [REFRESH MATERIALIZED VIEW](/sql/refresh-materialized-view) queries.
+  - `UPDATE`: [UPDATE](/sql/update) queries.
+  - `DELETE`: [DELETE](/sql/delete) queries.
+  - `ANALYZE`: [ANALYZE](/sql/analyze) queries.
+  - `DATA_DEFINITION`: Queries that affect the data definition. These include
+    `CREATE`, `ALTER`, and `DROP` statements for schemas, tables, views, and
+    materialized views, as well as statements that manage prepared statements, privileges, sessions, and transactions.
+  - `ALTER_TABLE_EXECUTE`: Queries that execute table procedures with [ALTER
+    TABLE EXECUTE](alter-table-execute).
+  - `MERGE`: [MERGE](/sql/merge) queries.
```

Comment:
MERGE should be moved to around INSERT, UPDATE, DELETE because they're similar operations. 

I suppose you used the same order as [QueryType](https://github.com/trinodb/trino/blob/master/core/trino-spi/src/main/java/io/trino/spi/resourcegroups/QueryType.java), but the order in the class is unrelated from user perspective.

Reply:
Thanks, @ebyhr. I'll make that adjustment.